packages/lib/src/vis/shared/VisMesh.tsx

```diff
@@ -1,18 +1,25 @@
 import type { MeshProps } from '@react-three/fiber';
 
+import type { Size } from '../models';
 import { useAxisSystemContext } from './AxisSystemContext';
 
-function VisMesh(props: MeshProps) {
-  const { children, ...meshProps } = props;
+interface Props extends MeshProps {
+  size?: Size;
```

Comment:
Same here: `width` and `height` ?

Reply:
Since it can be `undefined`, better keep a single object prop :+1: